src/classes/Construct.ts

```diff
@@ -6,12 +6,15 @@ import { CliOptions } from "../types/serverless";
  * Defines which methods a Lift construct must expose.
  */
 export interface ConstructInterface {
-    outputs(): Record<string, () => Promise<string | undefined>>;
+    /**
+     * Values shown in the CLI output.
+     */
+    outputs?(): Record<string, () => Promise<string | undefined>>;
```

Comment:
I've taken the opportunity to mark all functions as optional. Some constructs don't expose outputs or variables (just like commands), so that simplifies writing constructs.